AutonomousDemo/Velocity.cs

```diff
@@ -9,10 +9,10 @@ namespace AutonomousDemo
     public class Velocity
     {
         public double Speed { get; set; } = 0;
-        public double Direction { get; set; }  //radians
+        public double Direction { get; set; }  
```

Comment:
I dunno why you took off that comment, but you should put some kind of documentation specifying that direction is, in fact, radians. Maybe the fancy kind with three /'s.